### ISSUE TITLE
Links aren't rendering

### ISSUE THREAD

**Issue by kylebaron** [metrumresearchgroup/expo1-nonmem-foce#3]
Not sure what's wrong with the syntax. 

<img width="1371" alt="Screenshot 2023-01-04 at 1 42 08 PM" src="https://user-images.githubusercontent.com/16908301/210636238-fa50d298-53dd-4f4b-a242-6992c188b8bc.png">

**Comment by seth127** [comment]
It looks like we (I think me) forgot the `var` at the beginning. Here's a snippet of the source code:

```
see the {{< var pages.model_demo >}}. Guidance for creating model diagnostics are 
given on the {{< pages.model_diagnostics >}} and {{< pages.parameterized_reports >}} 
```

Notice `{{< var pages.model_demo >}}` (which works) vs. `{{< pages.model_diagnostics >}}` (which doesn't).

I'll verify that and push a fix. Thanks for spotting @kylebaron .

**Comment by seth127** [comment]
Ok this is fixed:

![Screenshot 2023-03-14 at 5 05 14 PM](https://user-images.githubusercontent.com/7806083/225137104-9d693799-b6d6-4909-b882-018206242eb1.png)

Just for traceability, this was fixed by commit `76f01bc63c86911c2471521e048a8522cf70455c` in the internal repo.
